backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/Info.java

```diff
@@ -35,13 +30,10 @@ private JsonNode getEndpointInfo(Schema schema) {
     }
     if (schema.getTable(ENDPOINT_TABLE) == null) return null;
 
-    Table table = schema.getTable(ENDPOINT_TABLE);
-
-    QueryBuilder queryBuilder = new QueryBuilder(table).addAllColumns(2);
-    GraphQL graphQL = new GraphqlApiFactory().createGraphqlForSchema(schema);
-    ExecutionResult result = graphQL.execute(queryBuilder.getQuery());
+    String query = "SELECT * FROM \"" + schema.getName() + "\".\"" + ENDPOINT_TABLE + "\"";
+    List<Row> rows = database.getSchema(schema.getName()).retrieveSql(query);
```

Comment:
is here also same permission issue? I suppose you might want to use Database.getJooqAsAdmin() (N.B. we should probably rename this) where you can use the wrapped transaction to do the elevated privileges thing.

Also you might want to use the 'Table' abstraction, however I do understand you don't need it here, this is shortest you could do.